svc/TrackService.js

```diff
@@ -10,8 +10,27 @@ import {stripTags} from 'hoist/utils/HtmlUtils';
 
 export class TrackService extends BaseService {
 
+    /**
```

Comment:
Would also like to have a class-level comment above, even though this single method clearly encapsulates much of the functionality.  Still worth it to take a few sentences and explain what user tracking is in general.  Feel free to take anything useful from hoist-core readme.  See ConfigService.js and PrefService.js for some examples (although those class-level comments are certainly longer than we would need for TrackService).

Part of what's driving this in my mind is that the JS classes might well be the first and primary source for documentation that a non-core Hoist developer will have readily available (in the form of the downloaded package).

Reply:
Added class level comment.